pyproject.toml

```diff
@@ -64,3 +64,18 @@ socks = [
 
 [tool.hatch.version]
 path = "src/urllib3/_version.py"
+
+[tool.hatch.build.targets.sdist]
+include = [
+  "/docs",
+  "/dummyserver",
+  "/src",
+  "/test",
+  "/dev-requirements.txt",
+  "/CHANGES.rst",
+  "/README.md",
+  "/LICENSE.txt",
+]
+exclude = [
+  "/docs/_build"
```

Comment:
No need since it's in https://github.com/urllib3/urllib3/blob/main/.gitignore

Reply:
Oooo, Hatch also reads .gitignore, interesting!

Reply:
Yup!

- https://hatch.pypa.io/latest/config/build/#vcs
- https://hatch.pypa.io/latest/config/build/#default-file-selection